Source/Evolve/WarpXEvolve.cpp

```diff
@@ -70,9 +70,6 @@ WarpX::Evolve (int numsteps)
     // Note that the default argument is numsteps = -1
     const int numsteps_max = (numsteps < 0)?(max_step):(istep[0] + numsteps);
 
-    bool early_params_checked = false; // check typos in inputs after step 1
```

Comment:
My own opinion is that since these flags are only used within this routine, making them class attributes is excessive and adds to bloat of the WarpX class.

Reply:
I thought so too, but here it is justified because our exit logic is non-trivial (and outlives the routine that checks). We can explore if a return type is easier.